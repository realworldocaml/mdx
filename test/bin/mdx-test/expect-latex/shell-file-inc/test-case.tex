Mdx can include text from any file:


% $MDX file=script.sh
\begin{sh}
\end{sh}
