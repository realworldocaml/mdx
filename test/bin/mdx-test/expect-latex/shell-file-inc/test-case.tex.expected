Mdx can include text from any file:


% $MDX file=script.sh
\begin{sh}
$ sudo chmod a-x /usr
\end{sh}
